org.mozilla.FirefoxESR.json

```
@@ -0,0 +1,1955 @@
+{
+    "app-id": "org.mozilla.FirefoxESR",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "firefox",
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc",
+        "--socket=x11",
+        /* Wayland access */
+        "--socket=wayland",
+        /* Needs to talk to the network: */
+        "--share=network",
+        /* Enable playing sound thru pulseaudio */
+        "--socket=pulseaudio",
+        /* Enable access to all devices (Mozilla's Firefox flatpak does this) */
+        "--device=all",
+        /* Enable themes support */
+        "--filesystem=~/.config/dconf:ro",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--talk-name=org.freedesktop.FileManager1",
+        /* Picked from Mozilla configuration */
+        "--socket=pcsc",
+        "--socket=cups",
+        "--persist=.mozilla",
+        "--filesystem=xdg-download:rw",
+        "--system-talk-name=org.freedesktop.NetworkManager",
+        "--talk-name=org.a11y.Bus",
+        "--talk-name=org.gnome.SessionManager",
+        "--talk-name=org.freedesktop.ScreenSaver",
+        "--talk-name=org.gtk.vfs.*",
+        "--talk-name=org.freedesktop.Notifications",
+        "--own-name=org.mpris.MediaPlayer2.firefox.*",
+        "--own-name=org.mozilla.firefox.*",
+        "--own-name=org.mozilla.firefox_esr.*",
+        "--own-name=org.mozilla.firefox_beta.*",
+        "--env=GTK_PATH=/app/lib/gtkmodules",
+        "--command=firefox",
+        "--allow=multiarch"
```

```suggestion
        "--env=GTK_PATH=/app/lib/gtkmodules"
```

You don't need either.